python/sglang/utils.py

```diff
@@ -127,9 +127,21 @@ def http_request(
     if api_key is not None:
         headers["Authorization"] = f"Bearer {api_key}"
 
+    if is_local:
+        if os.getenv("http_proxy") or os.getenv("https_proxy"):
```

Comment:
nit: to make the PR quickly merged, it would be great to keep the PR minimal i.e. only do bare minimal change related to video_reader-rs. And in a second separate PR the features like http proxy or port selection can be implemented.

(btw for proxy, we may need to consider HTTP_PROXY, HTTPS_PROXY etc)